packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -165,6 +167,13 @@ export interface ISpinButtonProps {
    * Accessibility label text for the decrement button for the benefit of the screen reader.
    */
   decrementButtonAriaLabel?: string;
+
+  /**
+   * To how many decimal places the value should be rounded to.
+   * The default value is calculated based on the precision of step.
+   * IE: if step = 1, precision = 0. step = 0.0089, precision = 4. step = 300, precision = 2. step = 23.00, precision = 2.
+   */
+  precision?: number;
```

Comment:
ITT either have this, or make step accept a string, but don't do both. It might make the most sense that step can just be used for precision but then again you have no way to check and make sure that the string I entered is valid.

Reply:
If you want step to be 200 and precision to be -1, there is no way to pass step as a string and satisfy that.